docs/04-Services/04-Navigation-Manager.md

```diff
@@ -0,0 +1,47 @@
+---
+since: 0.1.1
+---
+
+# Navigation Manager
+
+## Service Overview
+
+This service simplifies keyboard navigation for a set of DOM elements. It exposes a factory function `createNavManager()` that creates an object containing:
+
+- `elementsInDomOrder$` - a store containing the navigable elements in DOM order
+- `directive` - a directive to use on some DOM elements, both to add the keydown event handler and to specify which elements should be managed (either by directly putting the directive on those elements, or by putting the directive on a parent element and specifying which child elements should be included through a `selector` function via `NavManagerItemConfig`)
+- Utilities to navigate the items:
+  - `refreshElements` - method to manually refresh the `elementsInDomOrder$` to capture the latest state of DOM
+  - `focusIndex` - method to focus the element at the given index within `elementsInDomOrder$`
+  - `focusFirst` - method to focus the first element in the DOM order
+  - `focusLast` - method to focus the last element in the DOM order
+  - `focusPrevious` - method to focus the previous element in the DOM order
+  - `focusNext` - method to focus the next element in the DOM order
+  - `focusFirstLeft` - method to focus the element at the left-end of the list and will do the opposite in a RTL configuration
```

Comment:
For me the sentence is misleading regarding "the opposite". The opposite of "left-end" ? Does it mean "right-end" ? (of course not !)
I would prefer something like this:
```suggestion
  - `focusFirstLeft` - method to focus the element at the left-end of the list (identical to `focusFirst` in a LTR configuration and to `focusLast` in a RTL configuration)
```
(same remark for `focusFirstRight`, `focusLeft` and `focusRight`)

Reply:
makes sense :) fixed!